doc/Overview/Logging/LoggingOverview.md

```diff
@@ -1,108 +1,174 @@
 
+For more documentation about logging, read the references listed at the bottom.
 
 ## Platform Log Providers
 
-Wire up platform specific log providers (iOS: Uno.Extensions.Logging.OSLogLoggerProvider, WASM: Uno.Extensions.Logging.WebAssembly.WebAssemblyConsoleLoggerProvider), Debug and Console logging.
+To wire-up the platform specific log providers for debug and console logging, use the extension method `UseLogging()` on the `IHostBuilder` instance:
 
 ```csharp
-private IHost Host { get; }
+protected override void OnLaunched(LaunchActivatedEventArgs e)
+{
+    var appBuilder = this.CreateBuilder(args)
+        .Configure(host => {
+            host.UseLogging();
+        });
+...
+```
+
+## Logging
 
-public App()
+When the `UseLogging()` extension method is called, an `ILogger<T>` is registered in the service provider. This logger can be used to log messages to the console. It can then be injected into any class that needs to log messages. The `ILogger<T>` interface is generic, where `T` is the class that is logging the message. While the `ILogger<T>` interface is generic, it is not required to be used in this way. It can be used without the generic type parameter, but it is recommended to use the generic type parameter to scope the logger to a specific class.
+
+The library offers a number of extension methods to simplify the logging of messages for a specific log level. The following example shows how to log a message using the `LogInformation()` extension method:
+
+```csharp
+public class MyClass
 {
-    Host = UnoHost
-        .CreateDefaultBuilder()
-        .UseLogging()
-        .Build();
-    // ........ //
+    private readonly ILogger<MyClass> _logger;
+
+    public MyClass(ILogger<MyClass> logger)
+    {
+        _logger = logger;
+    }
+
+    public void MyMethod()
+    {
+        _logger.LogInformation("My message");
+    }
 }
 ```
 
-## Log levels
+## Log Levels
 
-We use the following convention for log levels:
+Sometimes it is necessary to write information to the log that varies in its verbosity or severity. For this reason, there are six log levels available to delineate the severity of an entry.
 
-  - **Trace** : Used for parts of a method to capture a flow.
-  - **Debug** : Used for diagnostics information.
-  - **Information** : Used for general successful information. Generally the default minimum.
-  - **Warning** : Used for anything that can potentially cause application oddities. Automatically recoverable.
-  - **Error** : Used for anything that is fatal to the current operation but not to the whole process. Potentially recoverable.
-  - **Critical** : Used for anything that is forcing a shutdown to prevent data loss or corruption. Not recoverable.
+Use the following convention as a guide when logging messages:
 
-```csharp
-private IHost Host { get; }
+| Level | Description |
```

Comment:
Where's this guidance taken from? Can we reference a good source for this, rather than it being made up?